CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ CHANGELOG
 # 0.10.0 - Next Release In Development
 
 * **BREAKING:** Using `.equalTo(<EdgeInsets>)` will no longer multiply the constants on the right/bottom edge by -1. This does not affect `.offset(<EdgeInsets>)`
```

Comment:
Hm good point, I'll revert this as it actually doesn't do what I thought it was going to.